frontend/i18n/de/projects/members.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  members: 'Mitglieder',
+  role: 'Rolle',
+  updateRole: 'Rolle aktualisieren',
+  addMember: 'Mitglied hinzufügen',
+  userSearchAPIs: 'Nutzer-Such-APIs',
+  userSearchPrompt: 'Tippen sie, um zu suchen',
```

Comment:
Sie

Reply:
You're mixing formal versions with informal ones (like in `label.js` or `home.js`). Stick to one. I think informal is better.